src/js/media-poster-image.js

```diff
@@ -41,10 +53,19 @@ const unsetBackgroundImage = (el) => {
 const setBackgroundImage = (el, image) => {
   el.style['background-image'] = `url('${image}')`;
 }
+const enableFade = (el) => {
+  el.classList.add('fade');
+}
+const disableFade = (el) => {
+  el.classList.remove('fade');
+}
+const loadFade = (el) => {
+  el.classList.add('loaded');
+}
 
 class MediaPosterImage extends window.HTMLElement {
   static get observedAttributes() {
-    return [MediaUIAttributes.MEDIA_HAS_PLAYED, 'placeholder-src', 'src'];
+    return [MediaUIAttributes.MEDIA_HAS_PLAYED, 'no-fade', 'placeholder-src', 'src'];
```

Comment:
Is `no-fade` a new attribute? Assuming we go this direction, I don't *think* we need it, honestly

Reply:
I agree, I'd prefer no added attribute but maybe possible to use a CSS var to override this fading functionality.

some users might prefer to see the image as soon as anything is loaded like if they use progressive jpeg format
https://medium.com/hd-pro/jpeg-formats-progressive-vs-baseline-73b3938c2339

Reply:
maybe if we expose `loading=lazy/eager` as on the `img` element, we can use that property to decide whether to fade or not. Where, with lazy, you probably want to fade, but with eager, you probably don't.